README.md

```diff
@@ -19,83 +19,115 @@ First, make sure your cluster fulfil the [Kubernetes package default requirement
 needs.
 
 ## Quickstart
 
-You are now ready to create a DC/OS cluster.
-
 Once the above pre-requisites have been met, clone this repo.
 
 ```
 git clone git@github.com:mesosphere/dcos-kubernetes-quickstart.git && cd dcos-kubernetes-quickstart
 ```
 
-Set your GCE credentials as environment variables. More information on how to obtain
-you credentials can be found [here](https://developers.google.com/identity/protocols/application-default-credentials)
+**For this Quickstart we are going to use GCE cloud provider**
 
-```
-export GOOGLE_APPLICATION_CREDENTIALS=<PATH TO YOUR CREDENTIAL FILE>
-```
+Install and setup Google Cloud SDK as per [doc](docs/gce.md).
 
-The remainder of this quick-start will execute in a Docker container, and create your cluster on GCE, with Kubernetes configured.  Simply run
+### Configure cluster
 
+Set GCP as cloud provider.
 ```
-$ make docker
+make gce
 ```
+The command above will download necessary [Terraform files](https://github.com/dcos/terraform-dcos/tree/master/gcp) to `.deploy` folder.
 
-You are now in a container from which you will deploy the cluster and required tools.
-
+Make updates to `.deploy/desired_cluster_profile` with your GCP `project-id` and `ssh key`, please do not change VMs to lover spec type, as then Kubernetes install will fail.
+```
+vi .deploy/desired_cluster_profile
+dcos_version = "1.10.2"
+num_of_masters = "1"
+num_of_private_agents = "3"
+num_of_public_agents = "1"
+#
+google_project = "YOUR_GCP_PROJECT"
+google_region = "us-central1"
+gce_ssh_pub_key_file = "PATH/YOUR_GCP_SSH_PUBLIC_KEY.pub"
+#
+gcp_bootstrap_instance_type = "n1-standard-1"
+gcp_master_instance_type = "n1-standard-8"
+gcp_agent_instance_type = "n1-standard-8"
+gcp_public_agent_instance_type = "n1-standard-8"
+# Inbound Master Access
+admin_cidr = "0.0.0.0/0"
 ```
-$ make deploy
-
-# Creates a ssh tunnel to a node-agent for APIServer access.
-$ make kubectl-tunnel
-# Make sure the API Server and Kubelets are up by running:
-$ kubectl get nodes
 
-# If you see a result like this, everything is working properly, and you are now running Kubernetes on DC/OS.
+For more cluster setup tweaks check out [here](https://github.com/dcos/terraform-dcos/tree/master/gcp).
 
-NAME                                   STATUS    AGE       VERSION
-kube-node-0-kubelet.kubernetes.mesos   Ready     3m       v1.7.10
-kube-node-1-kubelet.kubernetes.mesos   Ready     3m       v1.7.10
-kube-node-2-kubelet.kubernetes.mesos   Ready     3m       v1.7.10
+### Install command-line tools
 
-make uninstall
-# Uninstalls kubernetes.
-
-make destroy-dcos
-# Deletes the DC/OS cluster.
+Install DC/OS cli `dcos` and Kubernetes `kubectl`.
+```
+make get-cli
 ```
 
-## Installing DC/OS CLI
+Files `dcos` and `kubectl` will be downloaded to the current folder, please copy them for example, to `/usr/local/bin/`, or any other folder set in your `path`, so they can be invoked later one by install.
```

Comment:
If the user does move the binaries elsewhere it means that `make clean` won't actually remove them. This is confusing and somewhat poor UX. I'd rather have the user installing their own way and just make sure the versions are we expect them, just like we're doing with `terraform`.

Reply:
addressed not to remove dcos cli and kubectl